PhysicsTools/NanoAOD/python/genparticles_cff.py

```diff
@@ -6,7 +6,7 @@
 ##################### User floats producers, selectors ##########################
 
 finalGenParticles = cms.EDProducer("GenParticlePruner",
-    src = cms.InputTag("prunedGenParticles"),
+    src = cms.InputTag("mergedGenParticles"),
```

Comment:
ok I understand that with this you start with merged particles (pruned + packed), 
where pruned contain K* states, but not final states, and packed contain final states (K pi).
then you save also final states (granddaughters)

Reply:
Exactly